codegen.toml

```
@@ -104,7 +104,10 @@ output_dir = "java/lib/src/main/java/com/svix/models"
 template_dir = "codegen-templates/go"
 extra_shell_commands = [
    "rm go/health.go",
-   "sed -i 's/package svix/package internalapi/g' go/internalapi/management*"
+   # same thing as `sed -i 's/package svix/package internalapi/g' go/internalapi/management*`, but this will work both on macos(bsd-sed) and linux(gnu-sed)
+   "python -c \"import glob; [(lambda f: open(f, 'w').write(content))(f) for f in glob.glob('go/internalapi/management*') if (content := open(f, 'r').read().replace('package svix', 'package internalapi'))]\""
+
+
```

```suggestion
sed -i.bak 's/package svix/package internalapi/g' go/internalapi/management* && rm go/internalapi/management*.bak
```
If it's cursed, maybe switch to something not cursed?

---

> If it's cursed, maybe switch to something not cursed?

Good call. I will use this instead of the python monstrosity

> Also, why do we even need this sed?

Because I want to do something that the codegen won't let me, and this is easier then going into the codegen and re-architecting it